### PR TITLE
L2 750 cards container

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Show sns projects in the grid.
https://dfinity.atlassian.net/browse/L2-750

https://user-images.githubusercontent.com/98811342/176287827-7b1e03e1-1305-4e5c-b335-5f97e28f2434.mov

# Changes

- CardsGrid component
- move titles to the launchpad to apply styling in one place
- SNSProject -> SNSProjectCard

# Tests

- CardsGrid component

### CODE REVIEW

File: frontend/svelte/src/lib/components/sns-launchpad/SNSProjectCard.svelte [hunk 1]
```diff
@@ -51,19 +53,22 @@
   </dl>
 
   <!-- TODO L2-751: handle fetching errors -->
-  {#if swapState === undefined}
-    <Spinner size="small" inline />
-  {/if}
+  <div class="spinner">
```
Comment (lmuntaner) [inline frontend/svelte/src/lib/components/sns-launchpad/SNSProjectCard.svelte:56]: why should the div be always present?

File: frontend/svelte/src/lib/components/ui/CardGrid.svelte [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  <slot />
+</div>
+
+<style lang="scss">
+  @use "../../themes/mixins/media";
+
+  div {
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: var(--padding-4x);
```
Comment (lmuntaner) [inline frontend/svelte/src/lib/components/ui/CardGrid.svelte:11]: I created `--column-gap` for this purpose.

File: frontend/svelte/src/lib/components/ui/CardGrid.svelte [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    grid-auto-rows: minmax(30px, max-content);
```
Comment (lmuntaner) [inline frontend/svelte/src/lib/components/ui/CardGrid.svelte:12]: why 30px?

**Review by lmuntaner — CHANGES_REQUESTED** [review]
Minor change requested.

File: frontend/svelte/src/lib/components/ui/CardGrid.svelte (hunk 2 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/ui/CardGrid.svelte:11]: Not sure that this value will be used somewhere outside the component. I see it as a more strict structure 
```
CardsGrid
  Card  
  Card
```

File: frontend/svelte/src/lib/components/ui/CardGrid.svelte (hunk 3 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/ui/CardGrid.svelte:12]: Good point, updated with `0`

File: frontend/svelte/src/lib/components/sns-launchpad/SNSProjectCard.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/sns-launchpad/SNSProjectCard.svelte:56]: Fixed. Thanks!

File: frontend/svelte/src/lib/components/ui/CardGrid.svelte (hunk 2 above)
Comment (lmuntaner) [reply frontend/svelte/src/lib/components/ui/CardGrid.svelte:11]: ok

**Review by lmuntaner — APPROVED** [review]

**Pull request merged**